modules/minimint-wallet/src/keys.rs

```diff
@@ -28,16 +28,16 @@ impl MiniscriptKey for CompressedPublicKey {
         (*self).clone()
     }
 
-    fn serialized_len(&self) -> usize {
-        secp256k1::constants::PUBLIC_KEY_SIZE
-    }
+    // fn serialized_len(&self) -> usize {
+    //     secp256k1::constants::PUBLIC_KEY_SIZE
+    // }
```

Comment:
This we can remove this, too. Doesn't seem like it's used anywhere anymore ...

Reply:
removed in 'removed commented-out wallet code' commit